README.md

```diff
@@ -32,6 +33,33 @@ Read the [Changelog](https://github.com/Microsoft/vscode-go/blob/master/CHANGELO
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## We are moving!
+
+🏡 🚚 🏡 
+
+Our new home will be https://github.com/golang/vscode-go.
+
+As the extension grows in popularity and supports more features, it has required more maintenance and support.
+Ever since the introduction of Go modules, we have been collaborating with the Go team for module support.
+In 2019, we added support for `gopls`, the [Go language server](https://golang.org/s/gopls/README.md) from the
+Go team which has resulted in a closer collaboration. More exciting work is on the horizon with the
+[Go debug adapter service](https://github.com/go-delve/delve/tree/master/service/dap) in delve which will help
+in having a better debugging experience altogether.
+
+Moving to the [GitHub org for golang](https://github.com/golang) brings us closer to the home of other tools
+and libraries the Go extension depends on. We hope that this will help us remain abreast of relevant language
+changes and to encourage more gophers from the Go community to participate in the extension development.
+
+Over the next few weeks, we will work on the migration, including resolving open issues and PRs, and
```

Comment:
My proposed alternative for these final paragraphs:

Over the next few weeks, we will be working only on the migration--resolving and transferring open issues and PRs and updating documentation. Please follow our progress [here](http://TODO).

As of May 2020, this repository is effectively frozen. 
Only critical bug fixes or PRs related to the migration will be accepted.
You can still open new feature requests, bug reports, and PRs in the new repository: https://github.com/golang/vscode-go.

Reply:
I refactored this a little differently, checkout https://github.com/microsoft/vscode-go/pull/3237/commits/9af1f3c3c8e6274ea2288cc9af83a016d9695b8c and let me how that feels